src/cosp_stats.F90

```diff
@@ -273,10 +273,10 @@ END SUBROUTINE COSP_LIDAR_ONLY_CLOUD
   !               2018.05.17 (T.Michibata): update for COSP2
   !               2018.09.19 (T.Michibata): modified I/O
   !               2018.11.22 (T.Michibata): minor revisions
-  ! * References: Suzuki et al. (JAS'10, JAS'11, JGR'13, GRL'13, JAS'15)
+  !               2020.05.27 (T.Michibata and X.Jing): bug-fix for frac_out dimsize
+  ! * References: Michibata et al. (GMD'19, GRL'20)
```

Comment:
Are you able to provide DOIs for these papers?

Reply:
Thanks for this suggestion. I added DOI information for the (selected) references.